.github/workflows/coverage.yml

```
@@ -36,9 +36,8 @@ env:
 
 jobs:
   coverage:
-    name: Coverage nightly
+    name: Coverage on stable
     # The large timeout is to accommodate:
-    # - nightly builds (75 minutes, typically 30-50 minutes), and
     # - parameter downloads (an extra 90 minutes, but only when the cache expires)
     timeout-minutes: 165
```

Should we change this to 140 minutes ? 
```suggestion
    timeout-minutes: 140
```

---

Considered in [fix(ci): timeout and docs](https://github.com/ZcashFoundation/zebra/pull/4465/commits/9f6ddb7bb9082a8084b1933a9075f15a7cb41d04)